randovania/generator/generator.py

```diff
@@ -333,3 +345,25 @@ def actual_status_update(msg: str) -> None:
             )
 
     return result
+
+
+def force_logical_placement(
+    pickups: list[PickupEntry], game: GameDescription, lpp_config: LogicalPickupPlacementConfiguration
+) -> Requirement:
```

Comment:
Can you add some docstring for this function. See `_create_pools_and_fill` for an example.